package.json

```diff
@@ -36,6 +36,7 @@
     "url": "https://github.com/openlayers/openlayers/issues"
   },
   "dependencies": {
+    "eslint-config-openlayers": "^12.0.0",
```

Comment:
Is it correct to move the package from `devDependencies` to `dependencies` ?

Reply:
Thanks for the catch.  Fixed in deb00f20fe0c3b9bab4768d9cefe48c4de09a9fb.